integration/oasis/osrmconnector/osrmhttpclient.go

```diff
@@ -69,12 +83,19 @@ func (oc *osrmHTTPClient) response(m *message) {
 	defer close(m.req.tableRespC)
 
 	var routeResp RouteResponse
+	var tableResp TableResponse
 	if m.err != nil || m.resp == nil {
 		glog.Warningf("osrm request %s failed, err %v\n", m.req.url, m.err)
 
 		if m.req.t == OSRMRoute {
 			routeResp.Err = m.err
 			m.req.routeRespC <- routeResp
+		} else if m.req.t == OSRMTable {
+			tableResp.Err = m.err
+			m.req.tableRespC <- tableResp
+		} else {
+			glog.Error("osrmHTTPClient Only support OSRMRoute and OSRMTable request for now.")
```

Comment:
You can use `glog.Fatal` here.

Reply:
Good suggestion.  
```
Logging a FATAL message terminates the program (after the message is logged). 
```
And also get some useful information here: http://rpg.ifi.uzh.ch/docs/glog.html